<launch>

	<!-- machine -->
	<machine name="localhost" address="localhost" env-loader="/opt/ros/indigo/env.sh"/>
	<arg name="machine" default="localhost" />
  
	<arg name="parameters" default="constrained_whole_body_planner"/>
	
	<!-- load footstep hierarchical planner configurations from YAML file to parameter server -->
	<rosparam file="$(find dwl_planners)/config/$(arg parameters).yaml" command="load"/>	

	<node pkg="dwl_planners" type="constrained_whole_body_planner_node"
		  name="constrained_whole_body_planner" output="screen" machine="$(arg machine)">
<!--		<remap from="reward_map" to="/reward_map" />
		<remap from="obstacle_map" to="/obstacle_map" />
		<remap from="body_goal" to="/body_goal" />-->
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
<!--		<param name="world_frame" type="string" value="world" /-->
		<!-- Base frame of the robot -->
<!--		<param name="base_frame" type="string" value="base_link" /-->
	</node>
	
</launch>